csharp-mode.el

```diff
@@ -2460,6 +2460,14 @@ are the string substitutions (see `format')."
     ad-do-it)
   )
 
+(defadvice c-clear-string-fences (around
+                                  csharp-mode-advice-3
+                                  compile activate)
+  "Disable because it breaks csharp-mode when quoting. See
+https://github.com/josteink/csharp-mode/issues/151"
+  (if (c-major-mode-is 'csharp-mode)
+      ()))
```

Comment:
This line should be like the one in the advice above: Return either `nil` or `ad-do-it`.